[arduino_ide] Cannot build from source

```
compile:
    [mkdir] Created dir: /usr/local/tmp/crew/arduino-1.8.8.tar.xz.dir/arduino-1.8.8/app/bin
    [javac] Compiling 99 source files to /usr/local/tmp/crew/arduino-1.8.8.tar.xz.dir/arduino-1.8.8/app/bin
    [javac] /usr/local/tmp/crew/arduino-1.8.8.tar.xz.dir/arduino-1.8.8/app/src/processing/app/EditorToolbar.java:158: error: cannot access Window
    [javac]             touchBar.show(editor); 
    [javac]                     ^
    [javac]   class file for javafx.stage.Window not found
    [javac] Note: /usr/local/tmp/crew/arduino-1.8.8.tar.xz.dir/arduino-1.8.8/app/src/processing/app/Base.java uses or overrides a deprecated API.
    [javac] Note: Recompile with -Xlint:deprecation for details.
    [javac] Note: Some input files use unchecked or unsafe operations.
    [javac] Note: Recompile with -Xlint:unchecked for details.
    [javac] 1 error

BUILD FAILED
/usr/local/tmp/crew/arduino-1.8.8.tar.xz.dir/arduino-1.8.8/build/build.xml:136: The following error occurred while executing this line:
/usr/local/tmp/crew/arduino-1.8.8.tar.xz.dir/arduino-1.8.8/build/build.xml:113: The following error occurred while executing this line:
/usr/local/tmp/crew/arduino-1.8.8.tar.xz.dir/arduino-1.8.8/build/build.xml:161: The following error occurred while executing this line:
/usr/local/tmp/crew/arduino-1.8.8.tar.xz.dir/arduino-1.8.8/app/build.xml:82: Compile failed; see the compiler error output for details.

Total time: 11 seconds
arduino_ide failed to install: `env JAVA_HOME=/usr/local/share/jdk8 ant -Djava.net.preferIPv4Stack=true -Dversion=1.8.8 -Dplatform=linuxarm clean dist` exited with 1
```

--- a/packages/arduino_ide.rb
+++ b/packages/arduino_ide.rb
@@ -4,7 +4,7 @@ require 'package'
 
 class Arduino_ide < Package
 
-  @version = "1.8.8"
+  @version = "1.8.9"
 
   case ARCH
   when 'x86_64'
@@ -17,9 +17,9 @@ class Arduino_ide < Package
 
   description 'Arduino is an open-source physical computing platform based on a simple I/O board and a development environment that implements the Processing/Wiring language.'
   homepage 'https://www.arduino.cc/'
-  version '1.8.8'
+  version '1.8.9'
   source_url "https://github.com/arduino/Arduino/releases/download/#{@version}/arduino-#{@version}.tar.xz"
-  source_sha256 '05bbed313f12801ed37c5a38d642ec142ed8e7ccb34dd77815b45e663f1c0c7a'
+  source_sha256 '49e9c3a3a04c8dae8c2ffbfd39cf83b77d4908a09e29c2b3bfa4697c59ea1bf2'
 
   binary_url ({
   })

